docs/src/rules/no-tabs.md

```diff
@@ -12,26 +12,33 @@ This rule looks for tabs anywhere inside a file: code, comments or anything else
 
 Examples of **incorrect** code for this rule:
 
-::: incorrect
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable MD010 -->
+::: incorrect { "openInPlayground": false }
```

Comment:
```suggestion
::: incorrect
```

Reply:
Thanks, forgot to revert these changes. I guess it would be good to also validate parser options in rule examples.